editor.planx.uk/src/@planx/components/TextInput/Public.test.tsx

```diff
@@ -164,3 +167,54 @@ it("should change the role of the ErrorWrapper when an invalid input is given",
   expect(errorWrapper).not.toBeEmptyDOMElement();
   expect(errorWrapper).toHaveAttribute("role", "alert");
 });
+
+test("character limit counter should appear for long text inputs", async () => {
```

Comment:
testing follows a similar pattern to what I seen above, thought this homogeny would help readability.